features/works/work_download.feature

```diff
@@ -5,180 +5,172 @@
 Feature: Download a work
   @wip
```

Comment:
Any idea why these are marked with `@wip` and whether this is strictly necessary? I suppose once upon a time someone was running the tests manually using `~@wip" to exclude tests with this tag, but we don't do that anymore.